lectures/inflation_history.md

```diff
@@ -600,22 +643,22 @@ The US government stood ready to convert a dollar into a specified amount of gol
 
 Immediately after World War I, Hungary, Austria, Poland, and Germany were not on the gold standard. 
 
-Their currencies were  “fiat” or "unbacked",  meaning that they were not backed by credible government promises to convert them into gold or silver coins on demand.
+Their currencies were “fiat” or "unbacked", meaning that they were not backed by credible government promises to convert them into gold or silver coins on demand.
 
-The governments  printed new paper notes to pay for goods and services. 
+The governments printed new paper notes to pay for goods and services. 
 
 ```{note}
-Technically the notes were "backed" mainly by treasury bills. But people could not expect that those treasury bills would  be paid off by levying taxes, but instead by printing more notes or treasury bills.
+Technically the notes were "backed" mainly by treasury bills. But people could not expect that those treasury bills would be paid off by levying taxes, but instead by printing more notes or treasury bills.
 ```
- This was done on such a scale that it led to a depreciation of the currencies of spectacular proportions. 
+
+This was done on such a scale that it led to a depreciation of the currencies of spectacular proportions. 
  
- In the end, the German mark stabilized at 1 trillion ($10^{12}$) paper marks to the prewar gold mark, the Polish mark at 1.8 million paper marks to the gold zloty, the Austrian crown at 14,400 paper crowns to the prewar Austro-Hungarian crown, and the Hungarian krone at 14,500 paper crowns to the prewar Austro-Hungarian crown.
+In the end, the German mark stabilized at 1 trillion ($10^{12}$) paper marks to the prewar gold mark, the Polish mark at 1.8 million paper marks to the gold zloty, the Austrian crown at 14,400 paper crowns to the prewar Austro-Hungarian crown, and the Hungarian krone at 14,500 paper crowns to the prewar Austro-Hungarian crown.
 
-Chapter 3 of {cite}`sargent2002big`  described  deliberate changes in policy that Hungary, Austria, Poland, and Germany made to end their hyperinflations.
+Chapter 3 of {cite}`sargent2002big`  described deliberate changes in policy that Hungary, Austria, Poland, and Germany made to end their hyperinflations.
 
-Each governent  stoppped printing money to pay for goods and services once again made its currency  convertible  to the US dollar or the UK pound, thereby vitually  to gold.
+Each government stopped printing money to pay for goods and services once again which made its currency convertible to the US dollar or the UK pound.
```

Comment:
@jstac this is perhaps the most controversial edit.

Reply:
Thanks @mmcky .  I think your sentence is good but please change "which" to "and".